packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -559,6 +559,48 @@ describe("DropdownComponent", () => {
 					hostFixture.detectChanges();
 					expect(selectElement.outerHTML).toMatch(/ng-reflect-disabled="false"/);
 				});
+
+				it("Shouldn't trigger a 'valueChange' event when the formControl is disabled or enabled with emitEvent set to false", () => {
```

Comment:
could you please change this in lowercase as it is the case in the other tests ? 😊 
```
it("shouldn't trigger a 'valueChange' event when the formControl is di ...
```